javascript/builtins/Symbol.json

```diff
@@ -600,7 +600,8 @@
                 "version_added": "38"
               },
               "safari": {
-                "version_added": "10"
+                "version_added": "10",
+                "version_removed": "9"
```

Comment:
Symbol.species was probably not removed. Also version_removed can never be older than version_added. I will add an issue about this, so our tests catch this, too.